docs/src/main/tut/docs.md

```diff
@@ -13,33 +13,41 @@ title: "Documentation"
 
 ## Session creation
 
-**neotypes** adds an extension method (`.asScala[F[_]: Async]`) to `org.neo4j.driver.v1.Session` and `org.neo4j.driver.v1.Driver` that allows to build a `neotypes`'s session and driver wrappers.
-You can parametrize `asScala` by any type that you have a typeclass `neotypes.Async` implementation for.
-The typeclass implementation for `scala.concurrent.Future` is built-in _(please read more about [side effect implementations](docs/alternative_effects.html))_.
+**neotypes** provides the `GraphDatabase` **factory** for **Neo4j** `Session` and `Driver` wrappers.
+You can create wrappers for any type `F[_]` for which you have an implementation of the `neotypes.Async` **typeclass**.
+The implementation for `scala.concurrent.Future` is built-in _(for other types, please read [side effects](docs/alternative_effects.html))_.
```

Comment:
I'd replace "side effects" for just "effects" or "alternative effects".

Reply:
I think I prefer _"alternative effects"_. However, since this is documentation I will let the final vow to @dimafeng

Reply:
"alternative effects" sounds good to me